src/gpio.rs

```diff
@@ -678,24 +618,24 @@ macro_rules! gpio {
                         type Error = Infallible;
 
                         fn is_high(&self) -> Result<bool, Self::Error> {
-                            Ok(!self.is_low()?)
+                            Ok(unsafe { (*$GPIOX::ptr()).idr.read().$idri().is_high()})
```

Comment:
It's good practice to have for every use of `unsafe` a comment that explains why the code is safe. In this case you can just copy the one from `is_low`.

Reply:
I discarded my change and kept the previous code. So no comment is needed and I do not add more `unsafe` keywords :grinning: